plugins/themes/default/styles/objects/article_details.less

```diff
@@ -155,7 +155,8 @@
 				border-bottom: none;
 			}
 
-			.label {
+			h2.label {
```

Comment:
Sorry, I missed this in the last pass through. The `h2` should be removed here so that the selector is just `.label`. That way, if we have to change the heading level the styles will still apply.

Reply:
Just fixed this!